src/main/java/lotto/LottoController.java

```diff
@@ -0,0 +1,31 @@
+package lotto;
+
+import lotto.InputView;
+import lotto.ResultView;
+import java.util.List;
+
+public class LottoController {
+    private final InputView inputView;
+    private final ResultView resultView;
+    private final LottoMachine lottoMachine;
+
+    public LottoController(InputView inputView, ResultView resultView, LottoMachine lottoMachine) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+        this.lottoMachine = lottoMachine;
+    }
+
+    public void run() {
+        Money money = Money.of(inputView.readPurchaseAmount());
+
+        List<Lotto> lottos = lottoMachine.issue(money);
+        resultView.printLottos(lottos);
+
+        Lotto winning = new Lotto(inputView.readWinningNumbers());
+
+        WinningStatistics stats = new WinningStatistics();
+        stats.accumulate(lottos, winning);
```

Comment:
```suggestion
        Lotto winning = new Lotto(inputView.readWinningNumbers());

        WinningStatistics stats = winning.match(lottos);
```
위와 같이 구현하는 것과 현재와 같이 구현하는 것 중 어느 것이 나을까?
선택한 방법의 좋은 점은 뭘까?